app/src/main/java/com/example/dynamicmessenger/authorization/fragments/PersonRegistrationFragment.kt

```diff
@@ -95,12 +85,14 @@ class PersonRegistrationFragment : Fragment() {
             val name = binding.editTextName.text.toString()
             val lastname = binding.editTextLastname.text.toString()
             val username = binding.editTextUsername.text.toString()
-            val updateUserTask = UpdateUserTask(name, lastname, username, university, null,null,null,null,null)
+            val updateUserTask =
+                UpdateUserTask(name, lastname, username, university, null, null, null, null, null)
```

Comment:
You can have null as default values in order not to call constructor with so many params.